gravitee-apim-console-webui/src/management/api/portal/user-group-access/members/api-portal-members.component.ts

```diff
@@ -29,10 +30,18 @@ import { UsersService } from '../../../../../services-ngx/users.service';
 import { RoleService } from '../../../../../services-ngx/role.service';
 import { Role } from '../../../../../entities/role/role';
 import { GioPermissionService } from '../../../../../shared/components/gio-permission/gio-permission.service';
+import {
+  GioUsersSelectorComponent,
+  GioUsersSelectorData,
+} from '../../../../../shared/components/gio-users-selector/gio-users-selector.component';
+import { SearchableUser } from '../../../../../entities/user/searchableUser';
 
-class MembersDataSource extends ApiMember {
+class MembersDataSource {
```

Comment:
suggestion: rename to `MemberDataSource` (singular) (I know it was here before 😅)

Reply:
+1 i rename the both ;)